crates/ty_python_semantic/src/types/display.rs

```diff
@@ -611,12 +776,13 @@ impl DisplaySignature<'_> {
             let mut star_added = false;
             let mut needs_slash = false;
             let mut first = true;
+            let arg_separator = if multiline { ",\n    " } else { ", " };
```

Comment:
Does the indent work as expected for nested callable types (where a parameter type is a callable type itself that needs to break over multiple lines)

Reply:
This is avoided by the gratuitous uses of `settings.singleline()` preventing nested types from being line-wrapped almost anywhere.

Reply:
(I'll add tests to show that)